avro2tf/src/main/scala/com/linkedin/avro2tf/parsers/TensorizeInJobParamsParser.scala

```diff
@@ -39,7 +39,8 @@ case class TensorizeInParams(
   enableCache: Boolean,
   skipConversion: Boolean,
   outputFormat: String,
-  extraColumnsToKeep: Seq[String]
+  extraColumnsToKeep: Seq[String],
+  tensorsSharingFeatureLists: Option[Array[Array[String]]]
```

Comment:
should not defined this as optional, the default value defined at line 289 is enough, thus you can remove a lot of `case None => ()`

Reply:
Agreed. Will address.